Gemfile

```diff
@@ -50,5 +50,9 @@ group :development do
   gem 'spring-watcher-listen', '~> 2.0.0'
 end
 
+group :test do
+  gem 'rails-controller-testing'
```

Comment:
Cool, didn't know this existed!

Reply:
Yeah! It lets us verify that a template was rendered for a controller action for instance. Honestly, this project is really light so we might not even need to test for such things...